Src/java/build.gradle

```diff
@@ -111,14 +111,19 @@ subprojects {
     }
 }
 
-configure(subprojects.findAll {it.name in ['model', 'elm', 'quick', 'qdm', 'cql-to-elm']}) {
+/*
+JAXB dependencies:
+https://mkyong.com/java/jaxbexception-implementation-of-jaxb-api-has-not-been-found-on-module-path-or-classpath/
+ */
+
+configure(subprojects.findAll {it.name in ['model', 'model-jackson', 'model-jaxb', 'elm', 'elm-jackson', 'elm-jaxb', 'quick', 'qdm', 'cql-to-elm', 'cql-to-elm-cli']}) {
```

Comment:
Need to add the new projects to the publishing list down below.

Reply:
Added